lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -185,6 +185,22 @@ module.exports = [
           },
         },
       },
+      'efficient-animated-content': {
+        extendedInfo: {
+          value: {
+            wastedKb: 666,
+          },
+        },
+        details: {
+          items: [
+            {
+              url: 'http://localhost:10200/dobetterweb/lighthouse-rotating.gif',
+              totalBytes: 934285,
```

Comment:
👍 yeah @paulirish I told ward I'd tackle a more fine-grained assertion when I finish the other lantern opportunity rejiggering

Reply:
sgtm i like the look of this